### PR TITLE
Newsletter-2023-05-03-translate-in-French

### PR DESCRIPTION
**Pull request by Copinmalin**

### CODE REVIEW

**Comment by Copinmalin** [comment]
@bitschmidty , I think I need your help to fix the problem with ZH name please

File: _posts/fr/newsletters/2023-05-03-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+  apparemment deux fois la même transaction (voir [Bulletin #101][news101 overpayment]).
+  Cela a renforcé la nécessité de pouvoir inclure des transactions complètes
+  dans un PSBT.
+
+  Ce PR fusionné recherchera dans l'index des transactions (s'il est activé)
+  et dans le mempool du nœud local les transactions complètes, et les inclura
+  dans le PSBT si nécessaire. Si une transaction complète n'est pas trouvée,
+  l'ensemble UTXO sera utilisé pour les dépenses des sorties segwit. Notez
+  que Taproot (segwit v1) élimine le problème de surpaiement pour la plupart
+  des transactions qui dépensent au moins une sortie taproot, nous nous
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2023-05-03-newsletter.md:163]: The build is triggering on duplicate word here "nous nous". You can keep it (if it is correct to have it twice) but youll need to add "" at the beginning of this line.

Comment (Copinmalin) [reply _posts/fr/newsletters/2023-05-03-newsletter.md]: Ok, done for tips ""
but I still have the problem of missing files

Comment (bitschmidty) [reply _posts/fr/newsletters/2023-05-03-newsletter.md]: @Copinmalin sorry github stripped the previous comment. The duplicate line will need to be:

`<!-- skip-duplicate-words-test -->`

File: _posts/fr/newsletters/2023-05-03-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+  apparemment deux fois la même transaction (voir [Bulletin #101][news101 overpayment]).
+  Cela a renforcé la nécessité de pouvoir inclure des transactions complètes
+  dans un PSBT.
+
+  Ce PR fusionné recherchera dans l'index des transactions (s'il est activé)
+  et dans le mempool du nœud local les transactions complètes, et les inclura
+  dans le PSBT si nécessaire. Si une transaction complète n'est pas trouvée,
+  l'ensemble UTXO sera utilisé pour les dépenses des sorties segwit. Notez
+  que Taproot (segwit v1) élimine le problème de surpaiement pour la plupart
+  ""des transactions qui dépensent au moins une sortie taproot, nous nous
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2023-05-03-newsletter.md:163]:
```suggestion
  <!-- skip-duplicate-words-test -->des transactions qui dépensent au moins une sortie taproot, nous nous
```

**Review by bitschmidty — CHANGES_REQUESTED** [review]

File: _posts/fr/newsletters/2023-05-03-newsletter.md (hunk 1 above)
Comment (Copinmalin) [reply _posts/fr/newsletters/2023-05-03-newsletter.md]: Done !

File: _posts/fr/newsletters/2023-05-03-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], et
+[Bitcoin Inquisition][bitcoin inquisition repo].*
+
+- [Bitcoin Core #25158][] ajoute un champ `abandoned` aux réponses
+  détaillées des transactions des RPCs `gettransaction`, `listtransactions`,
+  et `listsinceblock` indiquant quelles transactions ont été marquées comme
+  [abandonnées][abandonransaction rpc].
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2023-05-03-newsletter.md:101]:
```suggestion
  [abandonnées][abandontransaction rpc].
```

**Review by bitschmidty — CHANGES_REQUESTED** [review]

**Comment by Copinmalin** [comment]
Yeah ! check pass 

Thanks @bitschmidty !

**Review by bitschmidty — APPROVED** [review]
ACK, thank you @Copinmalin !

**Pull request merged**